routes/blog.js

```diff
@@ -0,0 +1,9 @@
+const { Router } = require("express");
+const router = Router();
+
+// Controllers
+const controller = require("../controllers/blog");
+
+router.route("/").post(controller.create).get(controller.get);
```

Comment:
@umairian one of the best practice of REST API is resource naming convention, e.g it'll be good and make sense to both beginners and seniors if we make `createOne` and `getAll`.